projects/ngrx.io/content/guide/signals/signal-store/private-store-members.md

```diff
@@ -1,7 +1,7 @@
 # Private Store Members
 
 SignalStore allows defining private members that cannot be accessed from outside the store by using the `_` prefix.
-This includes root-level state slices, computed signals, and methods.
+This includes root-level state slices, properties, and methods.
```

Comment:
I think it can also be useful if the example below uses a private property using `withProps`